src/backends/odbc/statement.cpp

```diff
@@ -255,6 +255,25 @@ odbc_statement_backend::fetch(int number)
 
     for (std::size_t i = 0; i < fetchesCount; i += rowsPerFetch)
     {
+        if (fetchVectorByRows_) {
+            switch (intoType_)
+            {
+            case bt_standard:
+                // Standard intos processed in standard define_by_pos().
+                break;
+            case bt_vector:
+                // Unfortunately we ought to redefine all vector intos which
```

Comment:
```suggestion
                // Unfortunately we need to redefine all vector intos which
```
Also not sure why does it say "unfortunately"...